packages/device_info/device_info_platform_interface/lib/model/android_device_info.dart

```diff
@@ -21,57 +21,57 @@ class AndroidDeviceInfo {
     this.manufacturer,
     this.model,
     this.product,
-    List<String> supported32BitAbis,
-    List<String> supported64BitAbis,
-    List<String> supportedAbis,
+    List<String> supported32BitAbis = const [],
+    List<String> supported64BitAbis = const [],
+    List<String> supportedAbis = const [],
     this.tags,
     this.type,
     this.isPhysicalDevice,
     this.androidId,
-    List<String> systemFeatures,
+    List<String> systemFeatures = const [],
   })  : supported32BitAbis = List<String>.unmodifiable(supported32BitAbis),
         supported64BitAbis = List<String>.unmodifiable(supported64BitAbis),
         supportedAbis = List<String>.unmodifiable(supportedAbis),
         systemFeatures = List<String>.unmodifiable(systemFeatures);
 
   /// Android operating system version values derived from `android.os.Build.VERSION`.
-  final AndroidBuildVersion version;
+  final AndroidBuildVersion? version;
```

Comment:
For this one, and the ones below. It doesn't seem like they are set to `null` in the platform code: https://github.com/flutter/plugins/blob/42b33ec8ed3265607df5accf45bc68aa9e4e5592/packages/device_info/device_info/android/src/main/java/io/flutter/plugins/deviceinfo/MethodCallHandlerImpl.java#L39